addons/account_followup/account_followup.py

```diff
@@ -177,57 +143,55 @@ def do_partner_manual_action(self, cr, uid, partner_ids, context=None):
             else:
                 p = partner.latest_followup_level_id_without_lit.manual_action_responsible_id
                 responsible_id = p and p.id or False
-            self.write(cr, uid, [partner.id], {'payment_next_action_date': action_date,
-                                        'payment_next_action': action_text,
-                                        'payment_responsible_id': responsible_id})
+            self.write([partner.id], {'payment_next_action_date': action_date,
+                                      'payment_next_action': action_text,
+                                      'payment_responsible_id': responsible_id})
 
-    def do_partner_print(self, cr, uid, wizard_partner_ids, data, context=None):
+    def do_partner_print(self, wizard_partner_ids, data):
         #wizard_partner_ids are ids from special view, not from res.partner
         if not wizard_partner_ids:
             return {}
         data['partner_ids'] = wizard_partner_ids
-        datas = {
-             'ids': wizard_partner_ids,
-             'model': 'account_followup.followup',
-             'form': data
-        }
-        return self.pool['report'].get_action(cr, uid, [], 'account_followup.report_followup', data=datas, context=context)
+        datas = {'ids': wizard_partner_ids,
+                 'model': 'account_followup.followup',
+                 'form': data}
+
+        return self.env['report'].get_action([], 'account_followup.report_followup', data=datas)
 
-    @api.cr_uid_ids_context
-    def do_partner_mail(self, cr, uid, partner_ids, context=None):
-        if context is None:
-            context = {}
-        ctx = context.copy()
+    @api.multi
+    def do_partner_mail(self, partner_ids):
+        ctx = self.env.context.copy()
         ctx['followup'] = True
+        self.with_context(ctx)
         #partner_ids are res.partner ids
         # If not defined by latest follow-up level, it will be the default template if it can find it
-        mtp = self.pool.get('mail.template')
+        mtp = self.env['mail.template']
```

Comment:
Use CamelCase for model object !!!
Ex: 
MailTemplate = self.env['mail.template']